spyql/utils.py

```diff
@@ -32,3 +33,12 @@ def isiterable(x):
     from collections.abc import Iterable
 
     return isinstance(x, Iterable) and x is not Null and not isinstance(x, str)
+
+
+def folder(x):
+    """get the folder of this file path"""
+    return os.path.split(os.path.abspath(x))[0]
+
+def join(x, *args):
```

Comment:
```suggestion
def join_paths(x, *args):
```
some day we might have queries with `JOIN`s, and this would cause some confusion :-)